_emails/newsletter-2021-03.html

```diff
@@ -0,0 +1,148 @@
+---
+title: Origami Newsletter, March 2021
+companion_post_url: https://origami.ft.com/blog/2021/04/12/newsletter/
+---
+
+<!-- TL;DR -->
+{% capture text %}
+Updated Origami workshops. Work has begun to migrate Origami to npm. Reminder, we deprecate FT's Bower Registry in July. 2021 and plan to decomission in July 2022.
+{% endcapture %}
+
+{% include email/tldr.html content=text %}
+
+<!-- Top things -->
+{% include email/h2.html content="Top Things" %}
+
+{% capture text %}Some of the bigger Origami news from the last month:{% endcapture %}
+{% include email/markdown.html content=text %}
+
+
+
+<!-- Thing 1 -->
+{% include email/h3.html content="Update on the migration to npm (dropping Bower support)" %}
+{% capture text %}
+During January we announced our plan to [decommission the FT Bower Registry and migrate to the public npm registry](https://origami.ft.com/blog/2021/01/18/deprecating-bower-and-origami-via-npm/#who-does-this-affect).
+
+Work has now started on migrating the Origami components onto npm, 8 out of 66 of our component are now ready to be migrated. We aim to have all components, including documentation/guidance for how to migrate your products, complete by the start of Q3.
+
+As a reminder, below is a copy of our planned timeline to decommission the FT Bower Registry.
+
+1st July 2021 - Origami Components moved from Bower to npm and guides to migrate are written
+
+1st July 2021 - Deprecate the FT Bower Registry
+
+1st July 2022 - Decommission the FT Bower Registry
+
+{% endcapture %} {% include email/markdown.html content=text %}
+
+
+<!-- Thing 2 -->
+{% include email/h3.html content="New “Introduction to Origami components”workshop dates announced" %}
+{% capture text %}
+This workshop is great for people who are new to Origami or want a refresher on how to build with Origami.
+
+The workshop contents has had a revamp, it has been rewritten to be better suited for a remote audience and includes xyz and abc
+
+The dates we plan to run the workshop on are:
+- 42nd of borktober (include link to the Google Calendar entry)
+- 57th of Coffeeuary (include link to the Google Calendar entry)
+{% endcapture %} {% include email/markdown.html content=text %}
```

Comment:
Coffeeuary 😂 borktober 😂

I don't think I can include a calendar link without "publishing" the event, which would duplicate the event in individual calendars. I want to make sure we know who is coming and that we can update event details. Happy to share just the date here and post more reminders in Slack :) 

```suggestion
{% include email/h3.html content="New “Introduction to Origami” dates" %}
{% capture text %}
 
Please join us on 26th April for updated Introduction to Origami sessions! ✨ 

Part 1 is useful for anyone who works in Product and Technology – we cover things at a high level rather than diving into too many technical details, including:

- Origami History
- Origami Components
- Origami Services
- Origami Collaboration

Part 2 will explore components in more depth, including some technical details. It will be of particular interest to engineers and designers, although anyone is welcome:

- Documentation
- Languages & Tools
- Structure
- Versioning
- Origami Build Service / Manual Build
- Practical Tutorial (optional)

We'll be running these on the 26th April, at a time suitable for London and Sofia. We'll share a recording for those who can't make it, but it would be great to see you there! Please let us know if you or someone on your team would like an invite.

{% endcapture %} {% include email/markdown.html content=text %}
```